modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/model/impl/FriendlyURLEntryImpl.java

```diff
@@ -5,22 +5,70 @@
 
 package com.liferay.friendly.url.model.impl;
 
+import com.liferay.asset.kernel.model.AssetCategory;
+import com.liferay.asset.kernel.model.AssetEntry;
+import com.liferay.asset.kernel.service.AssetEntryLocalServiceUtil;
 import com.liferay.friendly.url.model.FriendlyURLEntry;
+import com.liferay.friendly.url.model.FriendlyURLEntryLocalization;
 import com.liferay.friendly.url.service.FriendlyURLEntryLocalServiceUtil;
+import com.liferay.petra.string.StringBundler;
 import com.liferay.petra.string.StringPool;
 import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
 import com.liferay.portal.kernel.util.LocalizationUtil;
 import com.liferay.portal.kernel.util.Validator;
 
 import java.util.Collection;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Map;
 
 /**
  * @author Pavel Savinov
+ * @author Roberto Díaz
  */
 public class FriendlyURLEntryImpl extends FriendlyURLEntryBaseImpl {
 
+	@Override
+	public String getCategorizedUrlTitle(String languageId) {
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-11147")) {
+			return getUrlTitle();
+		}
+
+		FriendlyURLEntryLocalization friendlyURLEntryLocalization =
+			FriendlyURLEntryLocalServiceUtil.fetchFriendlyURLEntryLocalization(
+				getPrimaryKey(), languageId);
+
+		if (friendlyURLEntryLocalization == null) {
+			return "";
```

Comment:
@adolfopa, I normally prefer StringPool.Blank, but there are cases where we return the empty string. Do we have any pattern for this?

Reply:
AFAIK, StringPool.BLANK is the preferred choice.